xyz.ayrsd.AudioStellar.yml

```diff
@@ -0,0 +1,319 @@
+      # - "-DSETCAP_OR_SUID_PING=false"
+      # - "-DSETCAP_OR_SUID_TRACEROUTE6=false"
+      - "-DUSE_GETTEXT=false"
+    sources:
+      - type: archive
+        url: https://github.com/iputils/iputils/archive/s20190709.tar.gz
+        sha256: a15720dd741d7538dd2645f9f516d193636ae4300ff7dbc8bfca757bf166490a
+
+  - name: freeglut
+    build-options:
+      cflags: -fcommon
+      cxxflags: -fcommon
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: "https://sourceforge.net/projects/freeglut/files/freeglut/3.2.1/freeglut-3.2.1.tar.gz"
+        sha256: d4000e02102acaf259998c870e25214739d1f16f67f99cb35e4f46841399da68
+        
+  - name: libxmu
+    sources:
+      - type: archive
+        url: https://www.x.org/archive//individual/lib/libXmu-1.1.3.tar.bz2
+        sha256: 9c343225e7c3dc0904f2122b562278da5fed639b1b5e880d25111561bac5b731
+        
+  - name: libraw1394
+    rm-configure: true
+    sources:
+      - type: archive
+        url: https://www.kernel.org/pub/linux/libs/ieee1394/libraw1394-2.1.2.tar.xz
+        sha256: 03ccc69761d22c7deb1127fc301010dd13e70e44bb7134b8ff0d07590259a55e
+      - type: script
+        commands: 
+         - autoreconf -fiv
+        dest-filename: autogen.sh
+      
+  - name: freeimage
+    buildsystem: simple
+    build-commands:
+      - sh gensrclist.sh
+      - sh genfipsrclist.sh
+      - sed -i 's/ throw (IEX_NAMESPACE::MathExc)//g' Source/OpenEXR/Imath/ImathVec.h
+      - sed -i 's/ throw (IEX_NAMESPACE::MathExc)//g' Source/OpenEXR/Imath/ImathVec.cpp
+      - sed -i 's/ throw (IEX_NAMESPACE::MathExc)//g' Source/OpenEXR/Imath/ImathMatrix.h
+      - make -f Makefile.gnu -j $FLATPAK_BUILDER_N_JOBS 
+      - make -f Makefile.fip -j $FLATPAK_BUILDER_N_JOBS
+      - make -f Makefile.gnu INCDIR=$FLATPAK_DEST/include INSTALLDIR=$FLATPAK_DEST/lib install
+      - make -f Makefile.fip INCDIR=$FLATPAK_DEST/include INSTALLDIR=$FLATPAK_DEST/lib install
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/freeimage/Source%20Distribution/3.18.0/FreeImage3180.zip
+        sha256: f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd
+      - type: shell
+        commands:
+          - sed -i 's/-o root -g root //g' Makefile.{gnu,fip}
+
+  - name: assimp
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DASSIMP_BUILD_ASSIMP_TOOLS:BOOL=NO
+      - -DASSIMP_BUILD_TESTS:BOOL=NO
+    sources:
+      - type: archive
+        url: https://github.com/assimp/assimp/archive/v4.1.0.tar.gz
+        sha256: 3520b1e9793b93a2ca3b797199e16f40d61762617e072f2d525fad70f9678a71
+
+  - name: rtaudio
+    config-opts:
+    - "--enable-shared"
+    - "--with-alsa"
+    - "--with-jack"
+    - "--with-pulse"
+    sources:
+    - type: archive
+      url: http://www.music.mcgill.ca/~gary/rtaudio/release/rtaudio-5.1.0.tar.gz
+      sha256: ff138b2b6ed2b700b04b406be718df213052d4c952190280cf4e2fab4b61fe09
+
+  - name: boost
```

Comment:
you need a cleanup rule to remove boost (expect the used DSO). the headers are not necessary in the final package.

Reply:
thanks, i'm not sure what to cleanup but i'll check it out. 

what do you mean by DSO ? 
are you saying this line is not necessary ? - ./b2 -j $FLATPAK_BUILDER_N_JOBS headers

thanks in advance !

Reply:
you already have some cleanup rules in the manifest so I assume you know what they do.

Doing what you suggest would break the build, or demonstrate that boost is not needed.

Reply:
https://docs.flatpak.org/en/latest/manifests.html#cleanup